test/e2e/warnings_test.go

```diff
@@ -72,27 +81,62 @@ spec:
   image: my-awesome-cron-image
 `
 
+	expectedOutputTemplate := `
+Target cluster 'https://127.0.0.1:60817' (nodes: minikube)
+
+Changes
+
+Namespace  Name  Kind     Conds.  Age  Op      Op st.  Wait to    Rs  Ri  
+kapp-test  <cr-name>  CronTab  -       -    create  -       reconcile  -   -  
+
+Op:      1 create, 0 delete, 0 update, 0 noop
+Wait to: 1 reconcile, 0 delete, 0 noop
+
+<replaced>: ---- applying 1 changes [0/1 done] ----<output-warning>
```

Comment:
Could have `<output-warning>` in the next line and use a backspace escape character for the case where warning is not shown? Might help if this is not legible enough